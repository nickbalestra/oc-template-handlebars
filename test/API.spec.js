const template = require('../packages/oc-template-handlebars');
const compiler = require('../packages/oc-template-handlebars-compiler');

describe('template APIs', () => {
  test('should expose the correct methods', () => {
    expect(template).toMatchSnapshot();
  });
});

describe('template-compiler APIs', () => {
  test('should expose the correct methods', () => {
    expect(compiler).toMatchSnapshot();
  });
});
