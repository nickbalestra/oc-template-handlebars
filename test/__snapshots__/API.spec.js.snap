// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template APIs should expose the correct methods 1`] = `
Object {
  "getCompiledTemplate": [Function],
  "getInfo": [Function],
  "render": [Function],
}
`;

exports[`template-compiler APIs should expose the correct methods 1`] = `
Object {
  "compile": [Function],
  "getCompiledTemplate": [Function],
  "getInfo": [Function],
  "render": [Function],
}
`;
